test/specs/json_ld/tests/02.json_ld_validator.js

```diff
@@ -0,0 +1,107 @@
+
+exports.execute = function (options) {
+    var module = options.includes.ermRest;
+    describe("validation of structured data", function () {
+        var jsonInputOrig = require("../resources/input_json_ld.json");
+
+        it("The google_metadata with incorrect attribute 'verson' returned w/o it", function (done) {
```

Comment:
I don't like how you're using the same object and deleting attributes or adding attributes. This spec is supposed to be self contained. So Just create a bare-minimum example of json-ld for each one and test based on that. You don't need a huge test file with long values and a lot of attributes.

Also write the description of `it` in a general tone. For example this one is `should ignore attributes that are not part of the vocabulary.`